### PR TITLE
Update Travel Expenses

### PR DESCRIPTION
**Pull request by codeekage**
So sorry for this; ran into an extra $90 expenses on Lodging. 🙏🏼

### CODE REVIEW

File: TravelFunds/2019.md [hunk 1]
```diff
@@ -20,5 +20,5 @@ Jory | Burson| Collaborator Summit | Attendance / Co-Organizing || Berlin | 28 M
 Anna | Henningsen | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 257.25€ | 29 Apr 2019 | ||286.45|26 Jun 2019|12 Jul 2019|$300
 Glenn | Hinks | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1660 | 25 Apr 2019 |||Withdrawn|N/A|N/A|N/A
 Mykola | Grybyk | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 320.57€ | 2 May 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
-Abraham | Agiri | OSCA (Open Soruce Africa) | Facilitator || Lagos NG | 1 - 4 August 2019 | $386.92 | 24 July, 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
-Richard | Trott | Node + JS Interactive 2019 | Collab Summit, presenting | | Montreal (Canada) | December 10 - 16 | $1750 | August 10 2019 | | | | | | 
\ No newline at end of file
+Abraham | Agiri | OSCA (Open Soruce Africa) | Facilitator || Lagos NG | 1 - 4 August 2019 | $476.72 | 24 July, 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
```
Comment (nschonni) [inline TravelFunds/2019.md]: Soruce -> Source?
```suggestion
Abraham | Agiri | OSCA (Open Source Africa) | Facilitator || Lagos NG | 1 - 4 August 2019 | $476.72 | 24 July, 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
```

**Comment by Trott** [comment]
> ran into an extra $90 expenses.

Lodging? Transportation? Something else?

**Comment by codeekage** [comment]
Denoted. 🙏

**Comment by Trott** [comment]
@nodejs/tsc @nodejs/community-committee Because this is an increase over the initial request, it will require the usual four approvals. It's a small amount compared to usual requests, and lodging should be an uncontroversial expense. Please consider adding your 👍! Thanks.

**Comment by codeekage** [comment]
@Trott please, can this land?

**Pull request merged**